clean_files.txt

```diff
@@ -95,6 +98,7 @@ test/test_helper.bash
 
 # themes
 #
+themes/*.bash
```

Comment:
This can be quite confusing, because this actually applies the rules on the base theme files and not on the themes themselves (they are in subdirectories). Maybe we should add a comment explaining this, or use explicit declaration here of each file

Reply:
I have a whole other thing about those files, but that's a different discussion that shouldn't be here. This line won't be in the PR when I mark it ready for review.